lib/config/presets/internal/group.ts

```diff
@@ -120,6 +121,15 @@ const staticGroups = {
       },
     ],
   },
+  codemirror: {
+    description: 'Group CodeMirror packages together',
+    packageRules: [
+      {
+        groupName: 'CodeMirror',
+        matchPackagePrefixes: ['@codemirror/'],
```

Comment:
Used `matchPackagePrefixes` instead of `matchSourceUrlPrefixes` because an older version has a separate repository (https://github.com/codemirror/CodeMirror), but it is not related to the `@codemirror` packages.